### ISSUE TITLE
Some image URLs cannot be displayed

### ISSUE THREAD

**Issue by vencia** [Renumics/spotlight#9]
Some Wikipedia images cannot be displayed, although the image URLs work in the browser.
E.g. first few images of the following dataset.
```
train_dset = load_dataset("alexandrainst/da-wit", split='train[:100]', data_files=['data/train-00000-of-00017-a976177c0f381298.parquet'], verification_mode=datasets.VerificationMode.NO_CHECKS)
df = train_dset.to_pandas()
spotlight.show(df, dtype={'image_url': spotlight.Image, 'embedding': spotlight.Embedding})
```

**Comment by druzsan** [comment]
I **confirm** this bug.

What I can deduce is that Wikipedia images do not load. If I enable dev mode, I get exceptions like
```
renumics.spotlight.dtypes.exceptions.InvalidFile: URL https://upload.wikimedia.org/wikipedia/commons/4/45/Bispen_-_inside.jpg does not exist.
```
and if I try manually:
```
import requests

response = requests.get("https://upload.wikimedia.org/wikipedia/commons/4/45/Bispen_-_inside.jpg")
```
I get not-ok response with error message including:
```
Error: 403, Forbidden. Please comply with the User-Agent policy: https://meta.wikimedia.org/wiki/User-Agent_policy
```
So I think this should help.
Working on this.

Please report, if some images other than from Wikipedia do not load too.

**Comment by druzsan** [comment]
BTW, everyone can enable debug mode through setting environment variable `SPOTLIGHT_DEV="true"` before strating CLI spotlight or python/notebook session to get rich exception outputs.

**Comment by druzsan** [comment]
Fixed in the version [1.0.0rc10](https://pypi.org/project/renumics-spotlight/)
